### DISCUSSION TITLE
vue3.3.4 use tailwindcss3.3.3 not result

### DISCUSSION THREAD

**Discussion by pythonliuwei-root** [tailwindlabs/tailwindcss#11721] [Help]
// my package.json
{
  "name": "ls",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "vue-tsc && vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "autoprefixer": "^10.4.14",
    "postcss": "^8.4.27",
    "tailwindcss": "^3.3.3",
    "vue": "^3.3.4"
  },
  "devDependencies": {
    "@vitejs/plugin-vue": "^4.2.3",
    "typescript": "^5.0.2",
    "vite": "^4.4.5",
    "vue-tsc": "^1.8.5"
  }
}
//my tailwind.config.js
/** @type {import('tailwindcss').Config} */
export default {
  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
  theme: {
    extend: {},
  },
  plugins: [],
};
// my main.ts
import { createApp } from 'vue'
import './style.css'
import App from './App.vue'

createApp(App).mount('#app')
//my App.vue

<template>

<div class="w-screen h-screen bg-slate-900 text-8xl">Hello world</div>

</template>
<script setup lang='ts'>
import {ref, reactive} from 'vue'
</script>
<style scopred>

</style>

**Comment by wongjn** [comment]
Could you elaborate more on problem you are experiencing please?
